src/shared-components/button/shared-components/loader/index.tsx

```diff
@@ -1,27 +1,27 @@
 import React from 'react';
+import { ThemeType } from 'src/constants/themes/types';
 
-import { COLORS } from '../../../../constants';
 import ButtonLoader from './style';
 import { ButtonTypeWithAction } from '../..';
 
 type LoaderProps = {
-  buttonColor?: string;
-  buttonType?: ButtonTypeWithAction;
+  buttonColor: valueof<ThemeType['COLORS']>;
+  buttonType: ButtonTypeWithAction;
   className?: string;
-  disabled?: boolean;
+  disabled: boolean;
   isFullWidth?: boolean;
-  isLoading?: boolean;
-  textColor?: string;
+  isLoading: boolean;
+  textColor: string;
 };
 
 const Loader = ({
```

Comment:
Loader is only used by Button components internally, so we can type things more strongly.